modules/economy.js

```diff
@@ -1,295 +1,255 @@
-      }
-      if (Object.prototype.hasOwnProperty.call(result, 0)) {
-        client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(result[0].amount) + plusNumber), member.id, guild.id], (err) => {
-          if (err) {
-            client.logError(err)
-            client.log.error(err)
-          }
-        })
-      } else {
-        client.pool.query('INSERT INTO `guildEconomyUserBank` (`member`, `guild`, `amount`) VALUES (?, ?, ?)', [member.id, guild.id, plusNumber], (err) => {
-          if (err) {
-            client.logError(err)
-            client.log.error(err)
-          }
-        })
-      }
+    module.exports.getMemberInventoryAndBalance(client, member, guild, (memberInventoryAndBalance) => {
+      client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(memberInventoryAndBalance.amount) + workMoney), member.id, guild.id], (err) => {
+        if (err) client.logError(err)
+        if (err) throw new Error('DB_ERROR')
+        if (callback && !err) callback(workMoney)
+      })
     })
-
-    if (callback) callback(plusNumber)
     EgM.finish()
   },
-  fetchConfig: async (client, guild, callback) => {
-    const EfC = client.Sentry.startTransaction({
-      op: 'economy.fetchConfig',
-      name: 'Economy (fetchConfig)'
-    })
-    client.pool.query('SELECT * FROM `guildData` WHERE guild = ?', [guild.id], (err, rows) => {
-      if (err) client.logError(err)
-      if (rows && Object.prototype.hasOwnProperty.call(rows, 0)) {
-        const data = { economyBankName: rows[0].economyBankName, economyBankLogo: rows[0].economyBankLogo, economyUseGlobalBank: rows[0].economyUseGlobalBank, economyCurrency: rows[0].economyCurrency }
-        callback(data)
-      } else {
-        callback(new Error('NO_DATA'))
-      }
-    })
-    EfC.finish()
-  },
-  fetchUserAccount: async (client, member, guild, callback) => {
+  getMemberInventoryAndBalance: async (client, member, guild, callback) => {
     const EfM = client.Sentry.startTransaction({
-      op: 'economy.fetchUserAccount',
-      name: 'Economy (fetchUserAccount)'
+      op: 'economy.getMemberInventoryAndBalance',
+      name: 'Economy (getMemberInventoryAndBalance)'
     })
     client.pool.query('SELECT * FROM `guildEconomyUserBank` WHERE guild = ? AND member = ?', [guild.id, member.id], (err, rows) => {
-      if (err) {
-        client.logError(err)
-        client.log.error(err)
-      }
+      if (err) client.logError(err)
+      if (err) throw new Error('DB_ERROR')
       if (Object.prototype.hasOwnProperty.call(rows, 0)) {
         callback(rows[0])
       } else {
-        module.exports.getDailyMoney(client, member, guild, false)
-        module.exports.fetchUserAccount(client, member, guild, callback)
+        client.pool.query('INSERT INTO `guildEconomyUserBank` (`member`, `guild`) VALUES (?, ?)', [member.id, guild.id], (err) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/alexandrumuntenas/pingu/pullRequest?prid=8657799)
```suggestion
        client.pool.query("INSERT INTO `guildEconomyUserBank` (`member`, `guild`) VALUES (?, ?)", [member.id, guild.id], (err) => {
```